backend/test/helpers/submission.js

```diff
@@ -1,18 +1,20 @@
-import { parseVideo } from '../../helpers/submittion'
 import { expect } from 'chai'
 
+import { parseVideo } from '../../helpers/submission'
+
 describe('Parse Video URL', function () {
-  it('accepts valid youtube urls', function () {
```

Comment:
we're not really doing anything asynchronous here, why have a `done`?

Reply:
Good catch! There was an unneeded `return Promise.resolve()` there before and my brain subconsciously changed it to `done()`. It's now removed.